etc/rc.local.openbsd

```diff
@@ -26,5 +26,4 @@ PYTHON=/usr/local/bin/python2.7
 # Don't change anything below here
 
 env PATH=${PATH}:/usr/local/bin sudo -u ${INITIAL_USER} sh -c \
-    "cd ${EXPERIMENT_DIR} && ${PYTHON} ${KRUN_DIR}/krun.py \
-   --resume --reboot --started-by-init ${CONFIG_FILE}" &
+    "cd ${EXPERIMENT_DIR} && ${PYTHON} ${KRUN_DIR}/krun.py --hardware-reboots ${CONFIG_FILE}" &
```

Comment:
We need to update the example `rc.local` files, but shall we do that as a separate PR?